code/__HELPERS/priority_announce.dm

```diff
@@ -23,6 +23,8 @@
 
 	announcement += "<br><span class='alert'>[html_encode(text)]</span><br>"
 	announcement += "<br>"
+	if(user)
+		announcement += "<span class='alert'>[user.name] ([user.job])</span><br>"
```

Comment:
should probably be preceeded by `-`

-Jeff Goldblum (Captain)

Reply:
Sounds good.

-Harrison Ford (Head of Personnel)